tests/src/Kernel/Export/ExportManagerTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Drupal\Tests\entity_sync\Kernel\Export;
```

Comment:
I'm not sure how this became part of this MR, it shouldn't be as we basically branched off of the MR that creates this test. Can you take a look and make sure.